packages/engine-rn-web/templates/appShell/index.html

```diff
@@ -1,34 +1,5 @@
 <html>
-
-<head>
-  <meta charset="UTF-8">
-  <title>App Shell</title>
-  <style>
-    body {
-      background-color: white;
-    }
-  </style>
-  {{APPSHELL_HTML_HEADER}}
```

Comment:
@mihaiblaga89 removing this inject APPSHELL_HTML_HEADER also makes config prop `hostedShellHeaders` no longer working. is that intentional?